src/Datadog.Trace/Conventions/OtelOutboundHttpConvention.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Tagging;
+using Datadog.Trace.Util;
+
+namespace Datadog.Trace.Conventions
+{
+    internal class OtelOutboundHttpConvention : IOutboundHttpConvention
+    {
+        private readonly Tracer _tracer;
+
+        public OtelOutboundHttpConvention(Tracer tracer)
+        {
+            _tracer = tracer;
+        }
+
+        public Scope CreateScope(OutboundHttpArgs args, out HttpTags tags)
+        {
+            tags = new OtelHttpTags();
+            var requestUri = args.RequestUri;
+            var httpMethod = args.HttpMethod;
+
+            string operationName = $"HTTP {httpMethod}";
+            string serviceName = _tracer.Settings.GetServiceName(_tracer, "http-client");
+            var scope = _tracer.StartActiveWithTags(operationName, tags: tags, serviceName: serviceName, spanId: args.SpanId);
+            scope.Span.Type = SpanTypes.Http;
+
+            tags.HttpMethod = httpMethod;
+            tags.HttpUrl = $"{requestUri.Scheme}{Uri.SchemeDelimiter}{requestUri.Authority}{requestUri.PathAndQuery}{requestUri.Fragment}";
+            tags.SetTag("http.scheme", requestUri.Scheme);
+            tags.SetTag("http.host", requestUri.Authority);
+            tags.SetTag("http.target", $"{requestUri.PathAndQuery}{requestUri.Fragment}");
```

Comment:
@zacharycmontoya  should I make it as `AdditionalTags` instead or would it be premature optimization?

Reply:
Either optimizing now or later is fine with me. If you think it would be a good example to demonstrate OTel-specific tagging, then that would be a good reason to do that now

Reply:
Done now.